lib/charms/layer/jenkins/api.py

```diff
@@ -128,15 +128,32 @@ def reload(self):
         self._execute_action(action, fail_message)
 
     def restart(self):
-        """Execute a safe restart. Wait for jobs, and restart"""
+        """Execute a safe restart. Wait for jobs, and restart."""
         hookenv.log("Waiting for all jobs to complete and restarting jenkins")
         action = "safeRestart"
         fail_message = "Couldn't restart jenkins"
         self._execute_action(action, fail_message)
         self.wait()
         unitdata.kv().set("jenkins.last_restart", time.time())
 
+    def quiet_down(self):
+        """Put Jenkins in a Quiet mode, in preparation for a restart.
+        In that mode Jenkins doesn’t start any build."""
```

Comment:
You'll want a blank line between these two lines per https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings